domain-server/src/DomainServer.cpp

```diff
@@ -66,6 +66,7 @@ using namespace std::chrono;
 
 Q_LOGGING_CATEGORY(domain_server, "hifi.domain_server")
 Q_LOGGING_CATEGORY(domain_server_ice, "hifi.domain_server.ice")
+Q_LOGGING_CATEGORY(domain_server_auth, "vircadia.domain_server.auth")
```

Comment:
Should we retain the "hifi." prefix and lodge an issue to fix up all logging categories to use "vircadia." instead?

Reply:
If we're going to do that it's probably better to go with 'vircadia.' here, since this way the mass-replace won't need to take this PR into account and will apply both before and after it.

I just checked, there's 78 instances of 'hifi.something' logging categories. Seems very doable, I can submit a PR for that.